SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -102,6 +102,10 @@ const Worker_ComponentId STARTUP_ACTOR_MANAGER_COMPONENT_ID = 9993;
 const Worker_ComponentId GSM_SHUTDOWN_COMPONENT_ID = 9992;
 const Worker_ComponentId HEARTBEAT_COMPONENT_ID = 9991;
 
+const Worker_ComponentSetId GDK_WELL_KNOWN_COMPONENT_SET_ID = 9999;
```

Comment:
I prefer not reusing component ids as component set ids (except where the component set is a wrapper for the component). Should make it clearer when debugging. Perhaps component sets start at 9000?

Reply:
I'm not a smart man